routes/habitRoutes.js

```diff
@@ -75,4 +75,22 @@ router.delete('/:id', passport.authenticate('jwt', { session: false }), (req, re
 		.catch(() => res.status(404).json({ message: 'Habit not found' }));
 });
 
+/**
+ * @route   PUT api/habit/:id
+ * @desc    Updates habit
+ * @access  Private
+ */
+router.put('/:id', passport.authenticate('jwt', { session: false }), (req, res) => {
+	const { name, description, type, difficulty, tags, frequency, user } = req.body;
```

Comment:
I guess that we will need some sort of validation here too, just check if incoming values are satisfying requirements from createHabit validator.